jerry-port/default/default-io.c

```diff
@@ -98,3 +98,32 @@ jerry_port_log (jerry_log_level_t level, /**< message log level */
     va_end (args);
   }
 } /* jerry_port_log */
+
+
+#ifdef JERRY_DEBUGGER
+
+#define DEBUG_BUFFER_SIZE (256)
+static char debug_buffer[DEBUG_BUFFER_SIZE];
+static int debug_buffer_index = 0;
+
+#endif /* JERRY_DEBUGGER */
+
+/**
+ * Default implementation of jerry_port_print_char. Uses 'printf' to
+ * print a single character to standard output.
+ */
+void
+jerry_port_print_char (char c) /**< the character to print */
+{
+  printf ("%c", c);
```

Comment:
Since the purpose of this function is to print a single character, would it make sense to use `putchar`? Could that make a difference?

Reply:
I'm not against it, currently I've just moved the original implementation. AFAIK if there is a `printf` on a port, the `putchar` should also be there. So it should not change anything. Should I change it to `putchar`?

Reply:
Both `printf` and `putchar` are good to me.

Reply:
I think `putchar` would be preferable here, but if you feel that it's outside of the scope of this change, then I won't insist.